helm/prometheus-rules/templates/alerting-rules/grafana.management-cluster.rules.yml

```diff
@@ -0,0 +1,50 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+    cluster_type: "management_cluster"
+  name: grafana.management-cluster.rules
+  namespace: {{ .Values.namespace }}
+spec:
+  groups:
+  - name: grafana
+    rules:
+    - alert: GrafanaDown
+      annotations:
+        description: '{{`Grafana ({{ $labels.instance }}) is down.`}}'
+        opsrecipe: grafana-down/
+      expr: up{service="grafana"} == 0
+      for: 15m
```

Comment:
Can we extend this to like either 30mn or 1h? 15mn feels very strict for something non-critical as Grafana.

Reply:
Fixed.